habitat/plan.sh

```diff
@@ -3,10 +3,17 @@
 #
 pkg_name=omnitruck
 pkg_origin=chef-es
-pkg_version="0.1.0"
+# Previous versions of omnitruck used an automatic version assignment for the
+# habitat pkg_version value which was base on the git commit count. Since we
+# use `sort --version-sort -r` in plan-build to determine the latest version of
+# any given package we need to set a value which sorts higher than the 950+
+# commits which are in the repo at the time of this change. Since we have 
+# additional hab services which depend on the omnitruck hab package, they will
+# uninintentionally install a version which is not the latest unless we use a
+# version which starts with 1000.x.x or greater.
+pkg_version=1000.0.0
```

Comment:
The other option is to continue bumping versions with the git commit count. Both suck. We went with the option that didn't require scripting...